src/ucp/wireup/select.c

```diff
@@ -800,6 +800,12 @@ static double ucp_wireup_am_bw_score_func(ucp_context_h context,
     return size / time * 1e-5;
 }
 
+static int ucp_wireup_is_ud_lane(ucp_context_h context, ucp_rsc_index_t rsc_index)
+{
+    return !strstr(context->tl_rscs[rsc_index].tl_rsc.tl_name, "ud") ||
+           !strstr(context->tl_rscs[rsc_index].tl_rsc.tl_name, "ud_mlx5");
```

Comment:
is this logic correct?  is this func need to return for 1  ud/ud_mlx5 ?  it that is that case, it seems it wrong logic

Reply:
I think it should be 's/strstr/strcmp'

Reply:
my fault :(
fix is on the way: https://github.com/openucx/ucx/pull/2235

Reply:
Thanks. I also added same fix in #2214  to fix the PR Jenkins